apps/src/code-studio/pd/workshop_dashboard/workshop_filter.jsx

```diff
@@ -313,23 +313,34 @@ export class WorkshopFilter extends React.Component {
     }));
   }
 
-  mergeSubjectArrays(objValue, srcValue) {
+  static getSubjectOptions(subjects, prefix = '') {
+    let result = {};
+
+    Object.keys(subjects).map(
+      course =>
+        (result[course] = subjects[course].map(subject => ({
+          value: subject,
+          label: prefix + subject
+        })))
+    );
+
+    return result;
+  }
+
+  static concatSubjetArrays(objValue, srcValue) {
```

Comment:
Subjet -> Subject

Reply:
hmm, I should enable spelling check on Rubymine